tools/rewriter/SourceRewriter.cpp

```diff
@@ -103,6 +103,10 @@ static Filename get_filename(const clang::SourceLocation loc,
       abort();
     }
   }
+  // if we could not get a filename for the spelling loc, use the expansion filename
```

Comment:
Would it make more sense to explicitly use `get_expansion_filename` at the callsites instead of silently falling back to the expansion loc? I haven't looked at the uses in detail enough to know which is cleaner.